authn-oidc-files/keycloak/create_user

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+keycloak/bin/kcadm.sh config credentials \
+  --server http://localhost:8080/auth \
+  --realm master \
+  --user "$KEYCLOAK_USER" \
+  --password "$KEYCLOAK_PASSWORD"
+keycloak/bin/kcadm.sh create users \
+  -s username=alice \
```

Comment:
Lets fetch this info from environment variable

Reply:
Done